presentation/common/src/test/hierarchy/HierarchyLevel.test.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import { HierarchyLevel } from "../../presentation-common/hierarchy/HierarchyLevel";
+import { Node } from "../../presentation-common/hierarchy/Node";
+import { createTestNode } from "../_helpers/Hierarchy";
+
+describe("HierarchyLevel", () => {
```

Comment:
I think we should decide how we format our tests moving forward. Because now we have two kinds of formatting for test:
1. This one in this file with a lot of empty lines before and after closing brackets.
1. [This one](https://github.com/iTwin/itwinjs-core/blob/e98bdf8e927d805c3d053572fb5edd446abebdbe/presentation/components/src/test/tree/Utils.test.ts#L120) without empty lines.

I would prefer the second one.

Reply:
I personally like the first approach better :) Filed [an issue](https://github.com/iTwin/itwinjs-core/issues/4940) so we could have a discussion, make the decision and update the existing tests.